app/views/fsss_view.py

```diff
@@ -92,7 +89,10 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
     data_job_role[mapField] = get_emp_info(empField)
   tab_job_role = tab_generation('Job Role for Field Worker', data_job_role)
 
-  data_contact = {}
+  data_contact = {
+      'Address': map_full_address_details(employee_info)
+      }
```

Comment:
The formatting on this is off- please re-run the formatter before each commit.

Reply:
Do you need to add a trailing comma to the end of the Address line? It's still indenting the ending bracket.